### ISSUE TITLE
Use with catfs on fstab?

### ISSUE THREAD

**Issue by eai04191** [kahing/goofys#346]
I'm using `goofys` and `catfs` with a command line, it works perfectly.
`$GOPATH/bin/goofys -o allow_other --dir-mode=0770 --uid=48 --gid=48 --cache "--free:50%:/tmp" some.bucket /mnt/s3/`

Next, I want to use auto mount. So I add a line at `/etc/fstab`.
```
/root/go/bin/goofys#some.bucket    /mnt/s3    fuse    _netdev,allow_other,--dir-mode=0770,--uid=48,--gid=48,--cache=--free:50%:/tmp 0 0
```

I run `mount -a`. but `goofys` said:
```
--cache requires catfs (http://github.com/kahing/catfs) but exec: "catfs": executable file not found in $PATH
```

How can I fix it? thanks.

**Comment by kahing** [comment]
is `catfs` in your $PATH?

**Comment by delia-m** [comment]
I have exactly same issue. 
I have catfs in $PATH and all works manual mount but for auto mount

**Comment by delia-m** [comment]
Same issue.. 
catfs in PATH (via .profile)
works when I mount manually with `--cache "--free:1%:/tmp"`
But got an error when I try to mount with `/etc/fstab`

**Comment by kahing** [comment]
is it in your login user's `PATH`, or root's `PATH`? automount via fstab is run via root, so it needs to be in root's `PATH`.

**Comment by delia-m** [comment]
Yes it's root's one. and I checked it's still there after logout and login again.
